src/openrct2/interface/Colour.h

```diff
@@ -212,31 +212,46 @@ constexpr uint8_t COLOUR_NUM_NORMAL = 54;
 #define TEXT_COLOUR_254 (254)
 #define TEXT_COLOUR_255 (255)
 
-enum : colour_t
+enum class ColourFlag : uint8_t
 {
-    COLOUR_FLAG_OUTLINE = (1 << 5),
-    COLOUR_FLAG_INSET = (1 << 6), // 64, 0x40
-    COLOUR_FLAG_TRANSLUCENT = (1 << 7),
+    translucent,
+    inset,
+    withOutline,
```

Comment:
I'd like to suggest we introduce a `none` type as well, and use that strictly instead of a `0` value. For example:
```suggestion
    none = 0,
    translucent = 1 << 0,
    inset = 1 << 1,
    withOutline = 1 << 2,
```

Reply:
Hmm, we do not have an equivalent of OpenLoco's EnumFlags.hpp yet, it seems. It might be too much hassle for this PR, then. Never mind.